includes/wc-template-functions.php

```diff
@@ -893,6 +893,10 @@ function woocommerce_template_loop_add_to_cart( $args = array() ) {
 
 			$args = apply_filters( 'woocommerce_loop_add_to_cart_args', wp_parse_args( $args, $defaults ), $product );
 
+			if ( array_key_exists( 'attributes', $args ) && array_key_exists( 'aria-label', $args['attributes'] ) ) {
```

Comment:
Any reason using this instead of `isset`? These could both be combined into a single isset() statement :)